### PR TITLE
CANS-540, FIT team Added health check

### PR DESCRIPTION
**Pull request by AVandana-OSI**
#### Which User story does this relate to (link)?
https://osi-cwds.atlassian.net/browse/CANS-540

#### Brief feature description
Added health check using the System Information GEM, which follows the standards as follows
https://github.com/ca-cwds/development-practices/blob/master/health_checks.md
which includes redis, Perry, CALS API, Dora API, GEO API were CALS dependent on.

#### Does this PR block/effect other stories?
No

#### Do any commands needs to be run in order to run the project with this PR?
No

#### Code coverage percent diff as a result of this PR?
-

### CODE REVIEW

**Comment by AVandana-OSI** [comment]
retest this please

**Comment by SashaKyz** [comment]
retest this please

**Comment by SashaKyz** [comment]
retest this please

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Make sure XCode is installed.
 ## Initialization
 
 run the command:
-`yarn run dev-build`
+`yarn dev`
```
Comment (edgibbs) [inline README.md]: So the command for `dev-build` still exists and the `dev` command runs foreman instead, so I'd need to have more context on whether it's the better default.

Comment (ratneshraval) [reply README.md]: `yarn dev` is better as it runs both rails and webpack dev server in hot mode.

Comment (AVandana-OSI) [reply README.md]: @edgibbs Yarn run dev-build was not starting the project locally, Yarn dev runs perfectly. Had a conversation with Nesh about the changing the read me file from `yarn run dev-build` to `yarn run` he is ok with it.

**Pull request merged**